tide_grant.module

```diff
@@ -114,6 +133,12 @@ function tide_grant_webform_submission_presave($submission) {
       'field_node_email' => $submission_data['contact_email_address'],
       'field_node_phone' => $submission_data['contact_telephone_number'],
     ]);
+    if ($has_sites) {
```

Comment:
This should be wrapped in a check for Tide Site.

Reply:
Thanks @GROwen , changed.